tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/promo_block.html

```diff
@@ -0,0 +1,19 @@
+{% load wagtailimages_tags %}
+
+<div class="grid__promo promo-block">
+    <div class="promo-block__content">
+        <h3>{{ value.title }}</h3>
+        <p>{{ value.description }}</p>
+        <a href="{{ value.get_button_link }}" class="button">{{ value.button_text }}</a>
+        {% if value.secondary_link %}
+            <hr>
+            {% with secondary_link=value.secondary_link.0.value %}
+                <a href="{{ secondary_link.url }}" class="promo-block__link">{{ secondary_link.text }}</a>
+            {% endwith %}
+        {% endif %}
+    </div>
+    {% image value.image fill-430x320 as mobile_image %}
+    {% image value.image fill-520x630 as desktop_image %}
+    <div class="promo-block__image promo-block__image--mobile" style="background-image: url('{{ mobile_image.url }}')"></div>
```

Comment:
As you have a quite different crop for mobile and desktop here, there could be a case for using a `picture` block here rather than `srcset_image` - see the listing image block for how we have done that: https://github.com/torchbox/torchbox.com/blob/integration/sprint-4/tbx/project_styleguide/templates/patterns/molecules/listing/listing--image.html

Reply:
> If it is because the image needs to fit the aspect ratio you can use object-fit:cover instead of background-size:cover
It was, `object-fit:cover` works nicely

Reply:
Resolved on the front-end ticket.